verl/trainer/ppo/ray_trainer.py

```diff
@@ -536,3 +567,10 @@ def fit(self):
             val_metrics = self._validate()
             pprint(f'Final validation metrics: {val_metrics}')
             logger.log(data=val_metrics, step=global_steps)
+
+
+@contextmanager
+def _timer(name: str, timing_raw: Dict[str, float]):
```

Comment:
It would be better to move this function above the `RayPPOTrainer`

Reply:
Sure

Reply:
Done